File: src/main/java/net/bms/genera/items/tab/GeneraTab.java
package net.bms.genera.items.tab;

import net.bms.genera.init.GeneraItems;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

public class GeneraTab extends CreativeTabs {
    public GeneraTab() {
        super("genera");
    }

    @Override
    public ItemStack getTabIconItem() {
        return new ItemStack(GeneraItems.ItemCinnabar);
    }
}

File: src/main/kotlin/net/bms/genera/item/tab/GeneraTab.kt
package net.bms.genera.item.tab

import net.bms.genera.init.GeneraItems
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack

class GeneraTab : CreativeTabs("genera") {

    override fun getTabIconItem(): ItemStack {
        return ItemStack(GeneraItems.ItemCinnabar)
    }
}
